torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -211,6 +218,9 @@ class TORCH_API TensorExprKernel {
   bool hasBroadcast_{false};
   std::unordered_map<const torch::jit::Value*, std::vector<ExprHandle>>
       known_sizes_;
+
+  std::vector<at::Tensor> unpacked_constant_tensors_;
+  std::vector<ConstantDescr> constants_;
```

Comment:
Do we prefer lowerCamelCase or lower_under_scores?  The existing members go both ways which makes me (╯°□°)╯︵ ┻━┻

Reply:
Agreed, maybe it's time to do some spring cleaning :) I've found pytorch cpp guidelines (unfortunately, the link is FB-internal): https://www.internalfb.com/intern/wiki/PyTorch/PyTorchDev/Workflow/Style/

The relevant part is this:

> NAMING
> * We follow the Facebook C++ Style Guidelines. Examples: some_namespace, SomeClass, SomeEnum, someFunction, someMethod, someMemberVariable, somePrivateMemberVariable_, SOME_MACRO
> * Exception: C++ methods and functions bound into python use python naming scheme (snake_case, ...) also in their C++ names. Exposing a previously unexposed method might need renaming the C++ method.
>